keyrings/alt/file.py

```diff
@@ -1,34 +1,36 @@
 from __future__ import with_statement
 
 import os
-import getpass
-import base64
 import sys
 import json
+import getpass
 
 from keyring.py27compat import configparser
 
 from keyring.util import properties
 from keyring.util.escape import escape as escape_for_ini
 
-from . import file_base
-
+from keyrings.alt.file_base import (
```

Comment:
I prefer the relative imports. It allows for shorter lines and easier migrations of code. Plus, it's less repetition. Plus, why not take advantage of the clearer context of the module name?

```
from . import file_base
```

Then use `file_base.Keyring` and `file_base.decodebytes` below. It'll be a nicer looking import section plus code below that's easier to recognize the context (without having to read the import section). Moreover, it gives the advantage of being able to leverage the namespace, so if you have code here that presents the same concept, it can re-use the name (such as where [setuptools overrides sdist](https://github.com/pypa/setuptools/blob/master/setuptools/command/sdist.py#L24)).